src/nvim/tui/term_input.inl

```diff
@@ -298,3 +286,10 @@ static void term_input_destroy(TermInput *input)
   event_poll(0);  // Run once to remove references to input/timer handles
   xfree(input);
 }
+
+static void term_input_set_encoding(TermInput *input, char* enc)
+{
+  int enc_flag = strcmp(enc, "utf-8") == 0 ? TERMKEY_FLAG_UTF8
+                                           : TERMKEY_FLAG_RAW;
```

Comment:
No, this is the exact string used by vim internally, regardless what form (`UTF8`, `UTF-8`, `utf8`, `unicode`, etc), happended to be in the env var or specified by the user.
Perhaps simpler, but more abstraction-leaky, would be to just pass a bool `is_utf8`. Or perhaps that's clearer actually because this PR already "breaks the abstraction" between TUI and GUI somewhat, what do you think?